source/common/tracing/zipkin/span_context.h

```diff
@@ -96,11 +70,24 @@ class SpanContext {
    */
   std::string traceIdAsHexString() const { return Hex::uint64ToHex(trace_id_); }
 
+  /**
+   * @return the sampled flag.
+   */
+  bool sampled() const { return sampled_; }
+
+  /**
+   * @return the sampled flag as a 1-character string.
```

Comment:
style choice, but the docs here make me feel like using a literal somehow and then just proving below with a unit test. I don't know if there is an ability to use a literal '1' and '0' in cpp, so if that's not possible ignore me.